### PR TITLE
backend assignment-1 done

### PR DESCRIPTION
**Pull request by nikhil1-successive**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const num1 = parseFloat(args[3]);
+const num2 = parseFloat(args[4]);
+const sumNumbers = add(num1, num2);
+const subtractNumbers = sub(num1, num2);
+const multiplyNumbers = mul(num1, num2);
+const divideNumbers = div(num1, num2);
+let result = 0;
+let operationFeasible = true
+
+if (operation === "Add") {
```
Comment (shahbaz-successive) [inline index.js:15]: Change name to operator symbols

File: OperationResultData.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Addition,2,30,32
```
Comment (shahbaz-successive) [inline OperationResultData.csv:1]: File should have xlsx extension and excel format is also not matching

**Review by shahbaz-successive — CHANGES_REQUESTED** [review]
Please do feedback changes

File: index.js (hunk 1 above)
Comment (nikhil1-successive) [reply index.js]: done

File: OperationResultData.csv (hunk 2 above)
Comment (nikhil1-successive) [reply OperationResultData.csv]: done

**Pull request merged**